apps/account-server/src/components/transaction-views/shared/TransactionIcon.tsx

```diff
@@ -21,6 +21,8 @@ export default function TransactionIcon({ transaction }: TransactionIconProps) {
   switch (transaction.transactionType) {
     case TransactionType.APPROVE:
       return <CoinsIcon weight="fill" className="w-10 h-10 text-white" />;
+    case TransactionType.ERC20:
```

Comment:
Nit comment. But y can just put the 2 cases on line after the other and return a single time for both